utils/src/main/scala/com/salesforce/op/utils/io/avro/AvroInOut.scala

```diff
@@ -182,12 +177,12 @@ object AvroInOut {
      *
      * @param path   Input directory where avro records should be written.
      * @param schema Avro schema string for records being written out.
-     * @param sc     Spark Session
      * @return
      */
-    def writeAvro(path: String, schema: String)(implicit sc: SparkSession): Unit = {
-      val jobConf = createJobConfFromContext(schema)
-      writeAvro(path)(jobConf)
+    def writeAvro(path: String, schema: String): Unit = {
+      val job: Job = Job.getInstance()
```

Comment:
how do we pass in hadoop configuration now? before it was picked up from spark context like this:
```scala
 val jobConf = new JobConf(sc.sparkContext.hadoopConfiguration)
```

Reply:
Since AvroWriter wraps the RDD it seems more correct to have a field
```scala
private implicit val job = Job.getInstance(rdd.sparkContext.hadoopConfiguration)
```
in AvroWriter

Reply:
excellent point @gerashegalov thanks for looking into it.